packages/core/src/App/Containers/RealAccountSignup/financial-details.jsx

```diff
@@ -96,10 +96,6 @@ const TradingExperience = ({
 class FinancialDetails extends React.Component {
     form = React.createRef();
```

Comment:
Can we remove this now that we removed references to it and replaced with `validateOnMount`?

Reply:
Absolutely. Will do.